src/Common/src/CoreLib/System/IO/FileStream.Win32.cs

```diff
@@ -19,8 +19,8 @@ private unsafe SafeFileHandle CreateFileOpenHandle(FileMode mode, FileShare shar
             Interop.Kernel32.SECURITY_ATTRIBUTES secAttrs = GetSecAttrs(share);
 
             int fAccess =
-                ((_access & FileAccess.Read) == FileAccess.Read ? GENERIC_READ : 0) |
-                ((_access & FileAccess.Write) == FileAccess.Write ? GENERIC_WRITE : 0);
+                ((_access & FileAccess.Read) == FileAccess.Read ? Interop.Kernel32.GenericOperations.GENERIC_READ : 0) |
```

Comment:
This isn't the best name- we should look at changing from `GenericOperations` to `GenericAccessRights` and add the [others](https://docs.microsoft.com/en-us/windows/win32/secauthz/generic-access-rights). Can do this in a separate change.

Reply:
I will do it in a separate change because the file already existed.

Reply:
I created this bug to track this: https://github.com/dotnet/corefx/issues/42199